sql/core/src/main/scala/org/apache/spark/sql/ExperimentalMethods.scala

```diff
@@ -46,4 +46,10 @@ class ExperimentalMethods private[sql]() {
 
   @volatile var extraOptimizations: Seq[Rule[LogicalPlan]] = Nil
 
+  def copy: ExperimentalMethods = {
```

Comment:
Changed to `clone()`. It does not seem straightforward to override `copy()` in case classes.